### ISSUE TITLE
Local files cannot be ignored permanently

### ISSUE THREAD

**Issue by msone** [languagetool-org/languagetool#7769]
Hello,

I use TiddlyWiki for private notes, which is an HTML file that can be edited over and over again. This file is locally on my hard disk. 

I can disable LanguageTool for the file, but that does not work permanently. Already in the next form LT is activated again. In the settings of LT under “Disable checking on specific websites” it says “file”.

To write sensitive content locally, I have to disable the plugin every time. Please fix the option.

System: macOS 13.2, Firefox 109.2, LanguageTool 6.0.1

**Comment by danielnaber** [comment]
How do your URLs look like? With files like `file:///home/dnaber/test.html` I cannot reproduce it, i.e. if I turn off checking for that, other files (like `file:///home/dnaber/test2.html`) will also be ignored.

**Comment by msone** [comment]
Yes, with `file:///`. Could you test it with a local tiddlywiki-file?

**Comment by danielnaber** [comment]
I don't use tiddlywiki, can you send an example file?

**Comment by msone** [comment]
You can download an empty file at: https://tiddlywiki.com/#GettingStarted 
When you have the file open in the browser, click on the "+" in the sidebar to create a new entry.  You can then see in the large text box at the bottom right that LT is still active.

**Comment by danielnaber** [comment]
Ok, I can reproduce this:

![Bildschirmfoto von 2023-01-30 14-59-51](https://user-images.githubusercontent.com/857004/215497692-7da9a30b-5462-4026-b16f-299bfb916a91.png)
